.gitignore

```diff
@@ -16,3 +16,4 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+*.sw[^f]
```

Comment:
I moved this to my global ignore (thought others using vim would appreciate it.) Will update shortly.

Would you like me to remove the other unnecessary ignores? I'd be happy to.